django_jsonform/validators.py

```diff
@@ -261,6 +261,7 @@ def validate_object(self, schema, data, coords, raise_exc=False):
                     next_schema['required'] = True
 
             next_type = get_schema_type(next_schema)
+            next_validator = None
```

Comment:
This looks good.

I noticed the `validate_array` method also has this same issue. If you could add this there as well (under Line 190), it would fix that method too.